### ISSUE TITLE
HumHub 1.15.0-beta.1: Error when disabling theme

### ISSUE THREAD

**Issue by felixhahnweilheim** [cuzy-app/humhub-modules-clean-theme#5]
HumHub: 1.15.0-beta.1
Clean Theme Module: 1.5.0

When I activate the clean-theme module and deactivate it right after, I get the following error.
Probably because module disabling is now done in a background job. So the theme views are still active but the module is not available.
The error disappeary after the background job has finished.

```
yii\base\ErrorException: Attempt to read property "collapsibleLeftNavigation" on null in /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/modules/clean-theme/themes/clean-base/views/ui/menu/widgets/views/left-navigation.php:25
Stack trace:
#0 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/modules/clean-theme/themes/clean-base/views/ui/menu/widgets/views/left-navigation.php(25): yii\base\ErrorHandler->handleError()
#1 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/View.php(347): require('...')
#2 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/View.php(257): yii\base\View->renderPhpFile()
#3 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/View.php(156): yii\base\View->renderFile()
#4 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/Widget.php(249): yii\base\View->render()
#5 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/humhub/modules/ui/menu/widgets/Menu.php(91): yii\base\Widget->render()
#6 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/humhub/components/Widget.php(63): humhub\modules\ui\menu\widgets\Menu->run()
#7 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/humhub/modules/admin/views/layouts/main.php(10): humhub\components\Widget::widget()
#8 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/View.php(347): require('...')
#9 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/View.php(257): yii\base\View->renderPhpFile()
#10 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/View.php(156): yii\base\View->renderFile()
#11 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/humhub/components/Controller.php(144): yii\base\View->render()
#12 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/Controller.php(408): humhub\components\Controller->renderContent()
#13 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/humhub/modules/admin/controllers/ModuleController.php(65): yii\base\Controller->render()
#14 [internal function]: humhub\modules\admin\controllers\ModuleController->actionList()
#15 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#16 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams()
#17 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction()
#18 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/web/Application.php(103): yii\base\Module->runAction()
#19 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/protected/vendor/yiisoft/yii2/base/Application.php(384): yii\web\Application->handleRequest()
#20 /var/www/vhosts/hahn-felix.de/dev.hahn-felix.de/index.php(25): yii\base\Application->run()
#21 {main} 
```

**Comment by felixhahnweilheim** [comment]
It seems to be a general problem for theme modules, see my comment: https://community.humhub.com/comment/perma?id=45889

**Comment by funkycram** [comment]
Thanks @felixhahnweilheim  for your investigation. I’ve reopened this issue: https://github.com/humhub/humhub/issues/5713#issuecomment-1663464004

**Comment by funkycram** [comment]
Will be fixed in next Humhub 1.5 version
